cmd/loadgenerator/main.go

```diff
@@ -57,13 +63,63 @@ func loadMnemonic(mnemonic string) crypto.Seed {
 	return seed
 }
 
+// Like shared/pingpong/accounts.go
+func findRootKeys(algodDir string) []*crypto.SignatureSecrets {
+	keylist := make([]*crypto.SignatureSecrets, 0, 5)
+	err := filepath.Walk(algodDir, func(path string, info fs.FileInfo, err error) error {
```

Comment:
nit, I think filepath.Walk wants you to do this with the err:
```suggestion
	err := filepath.Walk(algodDir, func(path string, info fs.FileInfo, err error) error {
	if err != nil {
		return err
	}
```